token-details-persister/src/main/scala/it/pagopa/interop/tokendetailspersister/Main.scala

```diff
@@ -17,7 +16,7 @@ object Main extends App {
 
   logger.info("Starting token details persister job")
   val fileManager: FileManager                 = FileManager.get(FileManager.S3)(blockingEC)
-  val dateTimeSupplier: OffsetDateTimeSupplier = OffsetDateTimeSupplierImpl
+  val dateTimeSupplier: OffsetDateTimeSupplier = OffsetDateTimeSupplier
```

Comment:
isn't there any use of the supplier where `get` needs to be replaced with `get()`?

Reply:
asefha;ebf;ouawebrpuiawbrpgiuaqbr

Angry since scalacOptions were overridden